src/canmatrix/formats/arxml.py

```diff
@@ -1153,12 +1162,12 @@ def get_signals(signalarray, frame, xmlRoot, ns, multiplex_id, float_factory):
               if l4 is not None:
                 Unit = l4.text
 
-        initvalue = arGetXchildren(syssignal, "INIT-VALUE/VALUE", xmlRoot, ns)
+        init_list = arGetXchildren(syssignal, "INIT-VALUE/VALUE", xmlRoot, ns)
```

Comment:
we need separate variable for list and string

Reply:
ok